### DISCUSSION TITLE
Support for pre-flight analysis of Microsoft Graph extensibility

### DISCUSSION THREAD

**Discussion by BernieWhite** [Azure/PSRule.Rules.Azure#3092] [Polls]
Support for defining a subset of Microsoft Graph resource using Bicep has been introduced as an experimental feature.

---

Since PSRule for Azure aligns to the Azure Well-Architect Framework (WAF), please include sample cases/ rules that would add value to the WAF alignment of PSRule for Azure.

**Comment by GaryRogersWoodmen** [comment]
I wonder if there's overlap with https://maester.dev/ here. It seems focused on testing these graph resources.

**Reply by BernieWhite** [reply]
Fair, for clarity. Support would be for IaC (pre-flight).
